### ISSUE TITLE
A way to queue songs

### ISSUE THREAD

**Issue by Merudo** [RPTools/maptool#966]
**Is your feature request related to a problem? Please describe.**
Once a song is done playing through playStream, the GM has to manually start a different song.

This is tedious and could be automated.

**Describe the solution you'd like**
An extra parameter to `playStream` so that it can queue songs, so that once one is done playing the next one starts.

**Describe alternatives you've considered**
An event triggered once a song stops playing, that could be used to start another song.

**Comment by melek** [comment]
I personally think an event is more flexible. That said, passing an array to playStream could also play a list of URLs in order of array index, maybe skipping any array contents that fail URL validation.

**Comment by Phergus** [comment]
I agree that an event might fit best with the current functionality.

**Comment by Maccwar** [comment]
Could playStream be tweaked or playStreams created to allow a JSON of multiple audio sources to be played in turn with associated metadata (volume, loops etc)?
